packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss

```diff
@@ -486,6 +489,113 @@ $Calendar-dayPicker-margin: 10px;
     }
   }
 
+  // Show the month picker over date picker (1 column layout), if enabled
+  .pickerIsMonthPickerOverlayVisible {
+    // Show the month picker.
+
+    .holder {
+      width: 230px;
+    }
+
+    .monthPicker {
+      display: block;
+    }
+    // Style the month and year pickers.
+    .monthPicker,
+    .yearPicker {
+      position: relative;
+    }
+    // Size and position of the month and year labels.
+    .month {
+      @include margin-left(12px);
+    }
+
+    .optionGrid {
+      width: 220px;
+      @include margin(10px, 0, 0, 0px);
+    }
+    // This component is only used on small displays.
+    .toggleMonthView {
+      display: none;
+    }
+    // Position the current year and decade labels.
+    .currentYear,
+    .currentDecade {
+      font-size: 17px;
+      margin: 0;
+      height: $Calendar-day;
+      line-height: 26px;
+      padding: 0;
+      display: inline-block;
+    }
+    // Reduce the size of the month buttons.
+    .monthOption,
+    .yearOption {
+      width: 40px;
+      height: 40px;
+      line-height: 38px;
+      font-size: 12px;
```

Comment:
Nitpick: some of these font sizes are already defined in sass variables. Additionally, some of the frequently repeated pixel values could be stored in new sass variables as well.